filters/diag/diag.go

```diff
@@ -417,3 +451,89 @@ func (t *throttle) Response(ctx filters.FilterContext) {
 	rsp := ctx.Response()
 	rsp.Body = t.goThrottle(rsp.Body, true)
 }
+
+func (j *jitter) Name() string {
+	switch j.typ {
+	case normalDist:
+		return "jitterNormal"
+	case uniformDist:
+		return "jitterUniform"
+	}
+	return "unknown"
+}
+
+func (j *jitter) CreateFilter(args []interface{}) (filters.Filter, error) {
+	var (
+		delay     time.Duration
+		interval  time.Duration
+		mu, sigma float64
+		ok        bool
+		err       error
+	)
+
+	if !(len(args) == 2 || len(args) == 4) {
+		return nil, filters.ErrInvalidFilterParameters
+	}
+
+	if delay, err = parseDuration(args[0]); err != nil {
+		return nil, fmt.Errorf("failed to parse duration delay %v: %w", args[0], err)
+	}
+
+	if interval, err = parseDuration(args[1]); err != nil {
+		return nil, fmt.Errorf("failed to parse duration interval %v: %w", args[1], err)
+	}
+
+	switch j.typ {
+	case normalDist:
+		switch len(args) {
```

Comment:
default is missing (e.g. len==3)

Reply:
so provide sigma as default?

Reply:
here it can only be 2 or 4, see https://github.com/zalando/skipper/pull/1804/files/2358921d50990ef3efa3da2240ad320e5b4fb1bf#diff-b091ac1989fb4f770f5e466781a6a12a59d432ec8f3ad1caaef8d8d2d245e950R474, but if we drop mu, then I can change it to 3